src/actionlist/action-list-tree.scss

```diff
@@ -15,6 +15,24 @@
 
 .ActionList--tree {
   --ActionList-tree-depth: 1;
+
+  // if direct sibling in ActionList--tree, and if not an expandable section (no chevron), indent to align with sibling icon or label
+  > .ActionList-item[aria-expanded] ~ .ActionList-item:not([aria-expanded]) {
+    .ActionList-content {
+      padding-left: $spacer-5;
+      background-color: palevioletred;
+    }
+  }
+
+  > .ActionList-item:not([aria-expanded]) {
```

Comment:
@vdepizzol am I missing an obvious selector pattern that will let me say..

`if direct child is not expandable and no future children are expandable, add padding`? Not possible right? Can't go down the chain and back up in one go. Otherwise this kinda works.